test/integration/etcd/etcd_storage_path_test.go

```diff
@@ -647,122 +640,56 @@ func TestEtcdStoragePath(t *testing.T) {
-			kubeAPIServerOptions.Authorization.Mode = "RBAC"
-
-			// always get a fresh port in case something claimed the old one
-			kubePort, err := framework.FindFreeLocalPort()
-			if err != nil {
-				t.Fatal(err)
-			}
-
-			kubeAPIServerOptions.SecureServing.BindPort = kubePort
-
-			tunneler, proxyTransport, err := app.CreateNodeDialer(kubeAPIServerOptions)
-			if err != nil {
-				t.Fatal(err)
-			}
-			kubeAPIServerConfig, sharedInformers, _, _, _, err := app.CreateKubeAPIServerConfig(kubeAPIServerOptions, tunneler, proxyTransport)
-			if err != nil {
-				t.Fatal(err)
-			}
-
-			kubeAPIServerConfig.APIResourceConfigSource = &allResourceSource{} // force enable all resources
-
-			kubeAPIServer, err := app.CreateKubeAPIServer(kubeAPIServerConfig, genericapiserver.EmptyDelegate, sharedInformers)
-			if err != nil {
-				t.Fatal(err)
-			}
-
-			kubeClientConfigValue.Store(kubeAPIServerConfig.GenericConfig.LoopbackClientConfig)
-			storageConfigValue.Store(kubeAPIServerOptions.Etcd.StorageConfig)
-
-			if err := kubeAPIServer.GenericAPIServer.PrepareRun().Run(wait.NeverStop); err != nil {
-				t.Log(err)
-			}
-
-			time.Sleep(time.Second)
-		}
-	}()
-
-	if err := wait.PollImmediate(time.Second, time.Minute, func() (done bool, err error) {
-		obj := kubeClientConfigValue.Load()
-		if obj == nil {
-			return false, nil
-		}
-		kubeClientConfig := kubeClientConfigValue.Load().(*restclient.Config)
-		// make a copy so we can mutate it to set GroupVersion and NegotiatedSerializer
-		cfg := *kubeClientConfig
-		cfg.ContentConfig.GroupVersion = &schema.GroupVersion{}
-		cfg.ContentConfig.NegotiatedSerializer = kapi.Codecs
-		privilegedClient, err := restclient.RESTClientFor(&cfg)
-		if err != nil {
-			// this happens because we race the API server start
-			t.Log(err)
-			return false, nil
-		}
-		// wait for the server to be healthy
-		result := privilegedClient.Get().AbsPath("/healthz").Do()
-		if errResult := result.Error(); errResult != nil {
-			t.Log(errResult)
-			return false, nil
-		}
-		var status int
-		result.StatusCode(&status)
-		return status == http.StatusOK, nil
-	}); err != nil {
+	kvClient, err := getEtcdKVClient(opts.Etcd.StorageConfig)
+	if err != nil {
 		t.Fatal(err)
 	}
 
-	kubeClientConfig := kubeClientConfigValue.Load().(*restclient.Config)
-	storageConfig := storageConfigValue.Load().(storagebackend.Config)
-
-	kubeClient := clientset.NewForConfigOrDie(kubeClientConfig)
-	if _, err := kubeClient.CoreV1().Namespaces().Create(&v1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: testNamespace}}); err != nil {
+	kubeClient := clientset.NewForConfigOrDie(restConfig)
+	_, err = kubeClient.CoreV1().Namespaces().Create(&v1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: testNamespace}})
+	if err != nil {
 		t.Fatal(err)
 	}
 
-	client, err := newClient(*kubeClientConfig)
-	if err != nil {
+	/*if _, err := kubeClient.CoreV1().Secrets(testNamespace).Create(&v1.Secret{
```

Comment:
The comment is intentional? Or debug/testing/dev hangover?

Reply:
fixing